ts/components/wallet/card/index.tsx

```diff
@@ -227,7 +228,9 @@ class CreditCardComponent extends React.Component<Props> {
                         CreditCardStyles.largeTextStyle
                       ]}
                     >
-                      {`${HIDDEN_CREDITCARD_NUMBERS}${item.pan.slice(-4)}`}
+                      {`${HIDDEN_CREDITCARD_NUMBERS}${getCardPan(item).slice(
```

Comment:
using a text field doesn't work very well, see ![here](https://user-images.githubusercontent.com/48206/42515272-aa4f81b0-845b-11e8-8d19-ec5fd4e11ff7.JPG)

Reply:
What would you recommend instead?

Reply:
I think the problem there is the font size, as you can see, also the `2001` is very big - you should probably find a way to scale the font size to make the text fit the width of the card (that looks also cut on the right)

Reply:
![image](https://user-images.githubusercontent.com/107715/42747961-a955ecc2-88de-11e8-8e3a-e12814adb74f.png)

Reply:
I'm not sure the font size is the problem here. I would say that the size of `2001` is the correct one (see pic below), but the "dots" there are showing up much larger than they should be, thus taking all the space (and the `2001` is not as scaled as the dots are, that's why I'm assuming it's not a font size problem). Can you provide some additional information on the device you were using here?


![image](https://user-images.githubusercontent.com/107715/42747979-b477d804-88de-11e8-8333-8e639953f432.png)

Reply:
sure, [this is](https://www.gsmarena.com/lg_leon-7053.php) the device

Reply:
I forgot to follow up on this. I have tried on devices with similar characteristics (same OS version and same screen size) but I was unable to replicate the issue. 

Since this is a UI-related problem, I would address this separately ( https://www.pivotaltracker.com/story/show/159231780 )